wallet/wallet.c

```diff
@@ -18,13 +18,28 @@
 
 static void outpointfilters_init(struct wallet *w)
 {
+	sqlite3_stmt *stmt;
 	struct utxo **utxos = wallet_get_utxos(NULL, w, output_state_any);
+	struct bitcoin_txid txid;
+	u32 outnum;
 
 	w->owned_outpoints = outpointfilter_new(w);
 	for (size_t i = 0; i < tal_count(utxos); i++)
 		outpointfilter_add(w->owned_outpoints, &utxos[i]->txid, utxos[i]->outnum);
 
 	tal_free(utxos);
+
+	w->utxoset_outpoints = outpointfilter_new(w);
+	stmt = db_prepare(w->db, "SELECT txid, outnum FROM utxoset WHERE spendheight = ?");
+	sqlite3_bind_null(stmt, 1);
```

Comment:
Binding here seems wierd: why not just say "WHERE spendheight IS NULL" directly?